inbc-program/README.md

```diff
@@ -178,11 +178,15 @@ INBC is only supporting the application update portion of AOTA.
 
 ### Usage
 ```
-inbc aota {--uri, -u=URI} 
-   [--app, -a APP_TYPE; default="application"] 
-   [--command, -c COMMAND; default="update"]
+inbc aota {--app, -a APP_TYPE} {--command, -c COMMAND}
+   [--uri, -u=URI]
+   [--version, -v VERSION]
+   [--containertag, -ct CONTAINERTAG]
+   [--file, -f FILE]
    [--reboot, -rb REBOOT; default="no"]
-   [--username, -un USERNAME] 
+   [--username, -un USERNAME]
+   [--dockerusername, -du DOCKERUSERNAME]
```

Comment:
We should add a note to mention that the user will be asked for the Docker password during the command execution.